docs/book/src/clusterctl/commands/alpha-rollout.md

```diff
@@ -0,0 +1,51 @@
+# clusterctl alpha rollout
+
+The `clusterctl alpha rollout` command manages the rollout of a cluster-api resource. It consists of several sub-commands which are documented below. 
+
+<aside class="note">
+
+<h1> Valid Resource Tpes </h1>
+
+Currently, only the following cluster API resources are supported by the rollout command:
+
+- machinedeployment
+
+</aside>
+
+### Restart 
+
+Use the `restart` sub-command to force an immediate rollout. For example, here the MchineDeployment `my-md-0` will be immediately rolled out:
```

Comment:
Should we add a note explaining what rollout means: deleting machines (including cordon/drain of workloads), recreating new ones according to maxUnavailable/maxSurge